aspnet/mvc/models/validation/sample/UsersController.cs

```diff
@@ -0,0 +1,18 @@
+using System.Linq;
+using Microsoft.AspNet.Mvc;
+using Microsoft.AspNet.Mvc.Rendering;
+using Microsoft.Data.Entity;
+using MVCMovie.Models;
+
+namespace MVCMovie.Controllers
+{
+    [Route("[controller]/[action]")]
+    public class UsersController : Controller
+    {       
+        [AcceptVerbs("Get", "Post")]
+        public IActionResult VerifyEmail(string email)
+        {
+            return Json(data: string.Format("/UsersController/VerifyEmail {0}.", email));
```

Comment:
This will cause the browser to display the given message to the user as a validation error. Should change this to perform actual validation, returning `true` when the value is good.